Dockerfile

```diff
@@ -68,26 +68,26 @@ RUN wget https://github.com/koalaman/shellcheck/releases/download/v$SHELLCHECK_V
     tar xJvf - shellcheck-v$SHELLCHECK_VERSION/shellcheck          && \
     mv shellcheck-v$SHELLCHECK_VERSION/shellcheck /bin             && \
     rmdir shellcheck-v$SHELLCHECK_VERSION
-RUN shellcheck -V
-
-# Set locale
-RUN locale-gen en_US.UTF-8
 
 # Print tool versions
-RUN bash --version | head -n 1
-RUN zsh --version
-RUN ksh --version || true
-RUN dpkg -s dash | grep ^Version | awk '{print $2}'
-RUN git --version
-RUN curl --version
-RUN wget --version
+RUN bash --version | head -n 1 \
+    && zsh --version \
+    && ksh --version || true \
+    && dpkg -s dash | grep ^Version | awk '{print $2}' \
+    && git --version \
+    && curl --version \
+    && wget --version \
+    && shellcheck -V
```

Comment:
Those layers basically has no overhead at all, no need to reduce the layer just because want to.

Also `shellcheck -V` should be right after its installation to make sure it installed with success.

Reply:
I can totally understand your point since no files are actually changed. Feel free to scrap it.